src/plugins/files/common/default_image_file_kind.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { FileKind } from './types';
+
+const id = 'defaultImage' as const;
+const tag = 'files:defaultImage' as const;
+const tags = [`access:${tag}`];
+const tenMebiBytes = 1024 * 1024 * 10;
+
+/**
+ * A file kind that is available to all plugins to use for uploading images
+ * intended to be reused across Kibana.
+ */
+export const defaultImageFileKind: FileKind = {
+  id,
+  maxSizeBytes: tenMebiBytes,
+  blobStoreSettings: {},
+  // tried using image/* but it did not work with the HTTP endpoint (got 415 Unsupported Media Type)
+  allowedMimeTypes: ['image/png', 'image/jpeg', 'image/gif', 'image/webp'],
```

Comment:
1. Let's also add `'image/avif'` 
2. 
> `'image/gif'`
Do we need this? 🤔 

3. How about `image/svg+xml`? Do we need this? Are there any edge cases?

Reply:
> Let's also add 'image/avif'

👍🏻 

> image/gif

I was thinking this could be a useful alternative before we support video streaming 😄 , but yeah gifs can be horrible for site performance.

> How about image/svg+xml? Do we need this? Are there any edge cases?

In conversation with @azasypkin I learned this can be a way to do XSS attacks, so I decided to leave it out for now